docs/plugins/introduction/structure.md

```diff
@@ -92,7 +92,7 @@ module.exports = () => ({
 });
 ```
 
-But that of course is not the only way to do it. Many people like the syntactic sugar and extensibility of classes
+But that of course is not the only way to do it. Many people like the syntactic sugar and extensibility of classes.
```

Comment:
Can optionally add a commas around `of course` since it is a parenthetical modifier, but it's not strictly required.